src/Autonomous.cpp

```diff
@@ -54,9 +86,40 @@ void Autonomous::autonomyIter() {
   //  output: movement command
   //
 
-  setCmdVel(0.5, 1.0);
+  if (arrived(gpsPose)) { //toPose(gps, currHeading)
+    std::cout << "arrived at gate" << std::endl;
+    std::cout << "x: " << gpsPose(0) << " y: " << gpsPose(1) << " theta: " << gpsPose(2) << std::endl;
+
+
+    // implement moving through gates
+    // then update target
+  } else {
+    double dtheta = pathDirection(lidar, gpsPose);
+    double speed = 10.0;
+    // TODO incredibly clever algorithms for state estimation
+    // and path planning and control!
+    
+    if(!lidar.empty()) {
+        speed = 5.0;
+    }
+    setCmdVel(dtheta, speed);
+  }
+}
+
+double Autonomous::pathDirection(points_t lidar, pose_t gpsPose) {
+    double dtheta;
+    if(lidar.empty()) {
+        dtheta = angleToTarget(gpsPose);
```

Comment:
`angleToTarget` returns an angle difference, whereas `setCmdVel` expects an angular _rate_. I think in this case the rover will still behave correctly, since this setting basically says we should turn so that we reach the target angle in one second. So this is just an FYI.